rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/RMController.java

```diff
@@ -104,38 +106,56 @@ public String getLogo350Url() {
         return RMImagesUnbundled.LOGO_350;
     }
 
-    /** if this is different than LOCAL_SESSION cookie, we need to disconnect */
+    /**
+     * if this is different than LOCAL_SESSION cookie, we need to disconnect
+     */
     private String localSessionNum;
 
-    /** periodically updates the local state */
+    /**
+     * periodically updates the local state
+     */
     private Timer updater = null;
 
-    /** periodically fetches runtime stats */
+    /**
+     * periodically fetches runtime stats
+     */
     private Timer statsUpdater = null;
 
-    /** remote gwt service */
+    /**
+     * remote gwt service
+     */
     private RMServiceAsync rm = null;
 
-    /** stores client data */
+    /**
+     * stores client data
+     */
     private RMModelImpl model = null;
 
-    /** shown when not logged in */
+    /**
+     * shown when not logged in
+     */
     private LoginPage loginPage = null;
 
-    /** shown when logged in */
+    /**
+     * shown when logged in
+     */
     private RMPage rmPage = null;
 
-    /** result of the latest call to {@link RMServiceAsync#getStatHistory(String, String, AsyncCallback)} */
+    /**
```

Comment:
Maybe it can be put back as one-line comment

Reply:
I will fix comments